sky/backends/backend_utils.py

```diff
@@ -1826,6 +1835,25 @@ def run_ray_status_to_check_ray_cluster_healthy() -> bool:
         return record
 
     # All cases below are transitioning the cluster to non-UP states.
+
+    if (not node_statuses and handle.launched_resources.cloud.STATUS_VERSION >=
+            clouds.StatusVersion.SKYPILOT):
+        # Note: launched_at is set during sky launch, even on an existing
+        # cluster. This could cause extra checks if the cluster was already up
```

Comment:
We have the `'status' == INIT` below and `not node_statuses` above, which means that there is no extra check if the cluster was already up?

Reply:
I mean if the cluster was already up, then sky launch was called and interrupted (leaving cluster in INIT).

Reply:
In that case, should not `node_statuses` contain status, and the condition above will not be true?

Reply:
Removed this part of the comment.